src/components/PlioQuestion.vue

```diff
@@ -119,6 +129,14 @@ export default {
 
     // Opens the question window
     openModal() {
+      // Show highlighted options if coming back to an answered question
+      // Wait 200 ms because it takes some time to find the DOM elements
+      if (this.plioQuestion.state == "answered") {
+        setTimeout(() => {
+          this.showResult();
+        }, 200);
+      }
+      
```

Comment:
When question pops up + and the user is returning to a previously answered question,
need to show what was the result at that time.

Settimeout because it takes about 100-200 ms (doesn't happen instantly) for JS to find the DOM elements.

Reply:
see if you can replace the `timeout` with a listener AND a timeout? 

Something like (but maybe not the same) as https://stackoverflow.com/questions/16149431/make-function-wait-until-element-exists